### DISCUSSION TITLE
Bug detected when deleting CRUD auto screens

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6325] [General]
From a postgreSQL table named base_article, I created the auto CRUD screens.
Then I deleted them.
Now if I re-generate the CRUD screens, the :id screen is called /base_article-2/:id instead of /base_article/:id
If i try to rename it, I get an error saying screen /base_article/:id already exists.

So I think that the deleting did not work properly, and somme system objects with the name /base_article/:id remain somewhere...

**Comment by melohagan** [comment]
Thanks @NicolasFabre81 I've noticed that myself in the past. I don't think it necessarily is directly a Postgres issue.

I've raised a bug here: https://github.com/Budibase/budibase/issues/6326

**Reply by NicolasFabre81** [reply]
After a F5 in the designer , I've been able to rename the screen.
